signs-work-order-map/.nvmrc

```diff
@@ -0,0 +1 @@
+20
```

Comment:
I _think_ can we go all the way up to `22`... at least `22` worked for my testing